modules/apps/content-dashboard/content-dashboard-document-library-test/src/testIntegration/java/com/liferay/content/dashboard/document/library/internal/item/filter/provider/test/FileAspectRatioContentDashboardItemFilterProviderTest.java

```diff
@@ -0,0 +1,238 @@
+import com.liferay.portal.kernel.search.Document;
+import com.liferay.portal.kernel.search.Field;
+import com.liferay.portal.kernel.search.SearchContext;
+import com.liferay.portal.kernel.search.SearchContextFactory;
+import com.liferay.portal.kernel.search.filter.BooleanFilter;
+import com.liferay.portal.kernel.search.filter.Filter;
+import com.liferay.portal.kernel.search.generic.BooleanQueryImpl;
+import com.liferay.portal.kernel.service.CompanyLocalService;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
+import com.liferay.portal.kernel.test.util.GroupTestUtil;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.test.util.ServiceContextTestUtil;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.FileUtil;
+import com.liferay.portal.kernel.util.MimeTypesUtil;
+import com.liferay.portal.kernel.util.WebKeys;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+import com.liferay.portal.search.legacy.searcher.SearchRequestBuilderFactory;
+import com.liferay.portal.search.searcher.SearchResponse;
+import com.liferay.portal.search.searcher.Searcher;
+import com.liferay.portal.test.rule.FeatureFlags;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.mock.web.MockHttpServletRequest;
+
+/**
+ * @author Mikel Lorza
+ */
+@FeatureFlags("LPD-30087")
+@RunWith(Arquillian.class)
+public class FileAspectRatioContentDashboardItemFilterProviderTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_group = GroupTestUtil.addGroup();
+	}
+
+	@Test
+	public void testGetContentDashboardItemFilter() throws Exception {
+		FileEntry squareAspectRatioFileEntry = _addDLFileEntry(
+			"dependencies/225x225.jpeg");
+		FileEntry tallAspectRatioFileEntry = _addDLFileEntry(
+			"dependencies/183x275.jpeg");
+		FileEntry wideAspectRatioFileEntry = _addDLFileEntry(
+			"dependencies/277x182.jpeg");
+
+		List<Document> documents = _getDocuments(_getMockHttpServletRequest());
+
+		Assert.assertEquals(documents.toString(), 3, documents.size());
+
+		_assertGetDocuments(squareAspectRatioFileEntry, "square");
+		_assertGetDocuments(tallAspectRatioFileEntry, "tall");
+		_assertGetDocuments(wideAspectRatioFileEntry, "wide");
```

Comment:
Why not use the AspectRatio enum here?

Reply:
Yes, I can do that.

Reply:
Hi @adolfopa , Analyzing it we can not use the AspectRatio enum here because is a private enum and belongs to content-dashboard-document-library-impl module, so we should  export it and so on and add the dependency of impl in our test module and we don't usually do that .

Sorry for that